Apps/W1/Email - Outlook REST API/app/src/EmailOutlookAPISetup.Page.al

```diff
@@ -268,6 +281,7 @@ page 4509 "Email - Outlook API Setup"
         UnsuccessfulTestMsg: Label 'We could not get access token with the current setup.\\Please verify the values on the page and try again.';
         SuccessfulTestMsg: Label 'Success! Your authentication was verified.';
         HiddenValueTxt: Label '******', Locked = true;
+        DefaultTenantId: Label 'Common', Locked = true;
```

Comment:
You were right to default to the value we are currently using, but now I'm thinking: maybe we should default to the actual tenant ID if we know the tenant (i. e. to `TenantInformation.GetTenantID()`). What do you think?

Reply:
I think that is not a problem, when opening the page "Email - Outlook API Setup" if the value is equal to "Common" try to get the TenantID with that command. By the way, for OnPrem customers, we should keep "common" as the default value and allow them to change it.

Reply:
I've update the functionality, making it use "Common" for OnPrem environments and the tenantID as suggeted for SaaS with TenantInformation.GetTenantID()